utils/find-overlay-dependencies.sh

```diff
@@ -68,16 +68,22 @@ for sdk in ${(k)SDKS}; do
   DEPENDS_ON=()
   printf "%s:\n\t" "$sdk"
   deps=$(echo "@import $1;" | xcrun -sdk $sdk clang -arch $arch -x objective-c - -M -fmodules 2>/dev/null)
+  if [[ $? != 0 ]]; then
+    # Don't `continue` here because we want to clear out the SDK line if it exists
```

Comment:
I switched to commenting out an unsupported platform line instead of deleting it. I'm pretty sure someone would see this in the git diff and it lets the script continue.